meta/main.yml

```diff
@@ -6,11 +6,13 @@
 # See also cisagov/skeleton-ansible-role#153.
 dependencies: []
 galaxy_info:
-  author: First Last
+  author: Shane Frasier
   company: CISA Cyber Assessments
-  description: Skeleton Ansible role
+  description: Install and configure systemd-resolved
   galaxy_tags:
-    - skeleton
+    - resolved
+    - systemd
+    - systemdresolved
```

Comment:
Just a thought (this should still comply with character restrictions).

```suggestion
    - systemd_resolved
```

Reply:
In fact this change does not comply with character restrictions:
```console
WARNING  Listing 1 violation(s) that are fatal
meta-no-tags: Tags must contain lowercase letters and digits only., invalid: 'systemd_resolved'
meta/main.yml:1

Read documentation for instructions on how to ignore specific rule violations.

             Rule Violation Summary              
 count tag          profile rule associated tags 
     1 meta-no-tags shared  metadata             

Failed: 1 failure(s), 0 warning(s) on 28 files. Last profile that met the validation criteria was 'safety'. Rating: 3/5 star
```

(I already tried it.)

Reply:
Classic `ansible-lint`!